vocode/streaming/synthesizer/azure_synthesizer.py

```diff
@@ -367,7 +367,10 @@ def get_message_up_to(
         ssml: str,
```

Comment:
my other suggestions require changing this type to `List[str]`

i'm pretty sure this isn't called very many places, so that should be fine. But if it's does break a lot of callsites, you can do `Union[str, List[str]]` and a runtime type check (if ssml is a string, use `[ssml]` instead)